_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,122 @@
+
+Here are some of the findings:
+
+- Only two of the seven projects were looked at: the 2006 Census Online and My Account, My Business Account projects met all of our audit criteria for well-managed projects.
+- Overall, the government has made limited progress since our last audit of IT projects in 1997. Although since 1998 the Treasury Board Secretariat has established a framework of best practices for managing IT projects, many of the problems we cited in past reports have persisted.
+- The quality of governance varied widely from project to project. In four of the seven projects we found that governance responsibilities were not carried out adequately because key issues that impacted project performance were either not reported or not resolved.
+- Five of the seven projects we looked at were allowed to proceed with a business case that was incomplete or out-of-date or contained information that could not be supported.
+- Four of the projects were undertaken by departments that lacked the appropriate skills and experience to manage the projects or the capacity to use the system to improve the way they deliver their programs.
+- Depending on the project, the quality of project management ranged from good to seriously flawed. In two cases, poor project management led to long delays and large cost overruns.
+
+[2011 AG report](https://www.oag-bvg.gc.ca/internet/English/parl_oag_201106_02_e_35370.html): Examination of the progress made from the 2006 recommendations
+
+Here are some of the findings:
+
+- Overall, the government has made unsatisfactory progress on its commitments in response to our 2006 recommendations. Although some improvements have been made, progress has been unsatisfactory in the important areas of governance and project business cases. Only two of the five projects we looked at, the Temporary Resident Biometrics Project and Integrated Revenue Collections systems, met most of our criteria for well-managed projects.
+- In order to increase the likelihood of success, departments have significantly reduced the scope of the projects and considerably extended their timelines. In the area of project governance, the Expenditure Management Information System and Global Case Management System have had important deliverables deferred without full analysis of the impacts and costs of not completing these projects. In three of the five projects examined, the project business cases did not identify measurable benefits or the benefits have not been measured. For example, the 2007 business case for the Secure Channel no longer quantifies the financial benefits.
+
+Once again these findings are a decade or more old and yet the issues pointed out are still relevant today. The underlying issue is slow response to change and a large lead time to be able to incorporate any change in requirements into projects and systems. One of the culprits of this is a large and complex release process which results in massive and risky releases.
```

Comment:
"Once again these findings are a decade or more old and yet the issues pointed out are still relevant today."

== To smooth out the wording ==>

"Unfortunately, after much effort these decade old findings are still prelevant today."